<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UO AnimUOP Analyzer v0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Animations found</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>AnimView</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="12" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Skip2 (compress?):</string>
          </property>
         </widget>
        </item>
        <item row="17" column="2">
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>Unknown2:</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>Unk1 (T.Stamp?):</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>BinFileHash:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1" colspan="7">
         <widget class="QLabel" name="label_binfile_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>AnimFile:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>DataBlockAddress:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>BinFile:</string>
          </property>
         </widget>
        </item>
        <item row="15" column="2">
         <widget class="QLabel" name="label_33">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Version:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>DecompressedSz:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>DataBlockLength:</string>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>Unknown3:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_skip1">
          <property name="text">
           <string>Skip1 (Adler32?):</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0" colspan="4">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>From Bin File (decompressed) data section</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="1" colspan="3">
         <widget class="QLabel" name="label_binfilehash_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>CompressedSize:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="3">
         <widget class="QLabel" name="label_compressedsize_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="10" column="3">
         <widget class="QLabel" name="label_decompressedsize_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="label_datablockaddress_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="12" column="3">
         <widget class="QLabel" name="label_skip2_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLabel" name="label_skip1_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="16" column="2">
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>AnimID:</string>
          </property>
         </widget>
        </item>
        <item row="18" column="2">
         <widget class="QLabel" name="label_38">
          <property name="text">
           <string>HeaderLength:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="4" rowspan="10" colspan="4">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>UOPFrameData</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item row="1" column="0" rowspan="2">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="6" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>yCenter:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Width:</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLabel" name="label_ycenter_val">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="5" column="3">
              <widget class="QLabel" name="label_width_val">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="4">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Frame Select</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_framecount_val">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_12">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>FrameCount: </string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_xcenter">
               <property name="text">
                <string>xCenter:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="label_frameid_val">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>Frame ID:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Frame number:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QSpinBox" name="spinBox_framenumber">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>99</number>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Direction:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QSpinBox" name="spinBox_direction">
               <property name="maximum">
                <number>4</number>
               </property>
              </widget>
             </item>
             <item row="6" column="3">
              <widget class="QLabel" name="label_height_val">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="label_xcenter_val">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Height:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="4">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Single Frame Data</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>DecompressedSz:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="8">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>Files Directory:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_browse">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_index">
            <property name="text">
             <string>Load anim table</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0" colspan="4">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Bin File Header</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="label_datablocklength_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QLabel" name="label_formatid_val">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>7</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QLabel" name="label_unknown1_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="18" column="3">
         <widget class="QLabel" name="label_headerlength_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QLabel" name="label_unknown3_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="16" column="1">
         <widget class="QLabel" name="label_bindecompressedsize_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="17" column="3">
         <widget class="QLabel" name="label_unknown2_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" colspan="7">
         <widget class="QLabel" name="label_animfile_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="15" column="3">
         <widget class="QLabel" name="label_version_val">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_31">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>FormatID:</string>
          </property>
         </widget>
        </item>
        <item row="16" column="3">
         <widget class="QLabel" name="label_animid_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" colspan="8">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_26">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Status:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_status_val">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>3</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Idle</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Animation ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_animid">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>2047</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Group ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_groupid"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
